std_msgs/Header header
bool left_validity
bool right_validity
geometry_msgs/Point left_eye_position
geometry_msgs/Point right_eye_position